get-data.js

```diff
@@ -24,5 +24,10 @@ const swagList = require('./data.json').map(swag => {
 
 	return swag;
 });
-
+//Sort list by most recent
+swagList.sort(function (a, b) {
+	a = new Date(a.dateAdded);
+	b = new Date(b.dateAdded);
+	return a > b ? -1 : a < b ? 1 : 0;
+});
```

Comment:
```suggestion
// Sort by newest opportunities
}).sort(({dateAdded: left}, {dateAdded: right}) => new Date(right) - new Date(left));
```

Reply:
I'm not sure if comitting this suggestion will mess up the file... Multiline comments is in beta, so this might be completely broken 😄